### DISCUSSION TITLE
More maintainers

### DISCUSSION THREAD

**Discussion by rdbisme** [mutationpp/Mutationpp#231] [General]
Ehi @jbscoggi, I was thinking this project might benefit from more maintaners. And I have also a candidate (@mgoodson-cvd). What do you think? 

Also @mgoodson-cvd would you be interested?

**Comment by mgoodson-cvd** [comment]
Yes, certainly. I've done a significant amount of development on my own fork of Mutation++. I think it is a very important code for the community and would like to help it continue to be developed and supported.

**Reply by rdbisme** [reply]
I've just sent you the invitation! Welcome on board :)

**Reply by mgoodson-cvd** [reply]
Got it. Thanks! Looking forward to contributing in more ways.

**Reply by rdbisme** [reply]
Ehi @mgoodson-cvd can you send me your email in private (https://rdb.is/me for contacts method). I'll add you as owner of the PyPI project since I want to deploy it to offical PyPI

**Reply by mgoodson-cvd** [reply]
@rdbisme I sent you an email, thanks.
